<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReBook</class>
 <widget class="QDialog" name="ReBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>4</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{border-width:2px;
border-color:gray;
border-radius:10px;
padding:2px;
border-style:outset;
}
QPushButton:pressed{
background-color: rgb(91, 85, 255);
}

QLineEdit{
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:gray;
padding:3px;
}

QDialog{
background-color: rgb(111, 166, 255);
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>741</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 32pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(56, 61, 59);</string>
   </property>
   <property name="text">
    <string>房间预定界面</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QLabel" name="dayNumsLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>入住天数</string>
   </property>
  </widget>
  <widget class="QLabel" name="roomImageLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>180</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="dayNumsBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentText">
    <string>1</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="endTimeLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endTimeEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="roomIdLabel">
      <property name="text">
       <string>  房间号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="rooIdEdit">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="userNameLabel">
      <property name="text">
       <string>  用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="RoomTypeLabel">
      <property name="text">
       <string>  身份证</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="carIdEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="roomPriceLabel">
      <property name="text">
       <string>电话号码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneIdEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="cardIdLabel">
      <property name="text">
       <string>房间类型</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RoomTypeEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="phoneIdLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>房间价格</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="roomPriceEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="startTimeEdit">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>331</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="startTimeLabel">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>359</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IsDebtBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>是否挂账</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReBook</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReBook</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
